static/js/util/api.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// @flow weak
```

Comment:
I believe it makes the file not throw errors when the type checker doesn't work out. I put it there temporarily because I just wanted to get your thoughts about the change below